Composer/packages/server/src/store/data.template.ts

```diff
@@ -10,6 +10,7 @@ import { BotProjectMetadata } from '../services/project';
 
 export default {
   version: 1,
+  templateFeedUrl: firstPartyTemplateFeed,
```

Comment:
this concerns me - as discussed, would prefer this to be absent unless customized to avoid zombie values if we change them later.  IS there a reason we can't do that? I believe all the other code is set up to use this default without it being present in the config

Reply:
Handled